cvat-ui/src/messages.tsx

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2023 CVAT.ai Corp
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import consts from 'consts';
+
+const { UPGRADE_GUIDE_URL } = consts;
+
+const SERVER_UNAVAILABLE = (
+    <>
+        Make sure the CVAT backend and all necessary services
+        (Database, Redis and Open Policy Agent) are running and avaliable.
+        If you upgraded from version 2.2.0 or earlier, manual actions may be needed,
+        see the&nbsp;
+        <a
+            target='_blank'
+            rel='noopener noreferrer'
+            href={UPGRADE_GUIDE_URL}
+        >
+            Upgrade Guide
+        </a>
+        .
+    </>
+);
```

Comment:
If we want to override the message for app.cvat.ai, how would it work with this file?

Reply:
The idea is to build UI image with a custom messages.tsx file... this is not a ideal solution but I can't find another approach that can be quickly implemented